### PR TITLE
Update with transactions

### PR DESCRIPTION
**Pull request by Jeanbulambo**
In this PR, we implemented the following elements :+1: 
- Added a column species of type string to your animal's table. Modify your schema.sql file.
- Inserted data into the animal's table
- Made transactions
transaction
transaction 2
transaction 3
transaction 41
transaction 42

### CODE REVIEW

File: schema.sql [hunk 1]
```diff
@@ -7,3 +7,5 @@ CREATE TABLE animals(
     neutered BOOLEAN NOT NULL,
     weight_kg DECIMAL NOT NULL
 );
+
+ALTER TABLE animals ADD COLUMN species VARCHAR(50);
```
Comment (mateo951) [inline schema.sql:11]: add a new column called species 🙌

File: data.sql [hunk 2]
```diff
@@ -11,4 +11,24 @@ VALUES
...
+    weight_kg
+  )
+VALUES
+(005, 'Charmand', '2020-02-08', 0, false, -11),
+  (006, 'Plantmon', '2021-11-15', 2, true, -5.7),
+  (007, 'Squirtle', '1993-04-02', 3, false, -12.13),
+  (008, 'Angemon', '2005-06-12', 1, true, -45),
+  (009, 'Boarmon', '2005-06-07', 7, true, 20.4),
+  (010, 'Blossom', '1998-10-13', 3, true, 17),
+  (011, 'Ditto', '2022-05-14', 4, true, 22);
```
Comment (mateo951) [inline data.sql:34]: Propper data was introduced 😎

File: queries.sql [hunk 3]
```diff
@@ -23,3 +23,65 @@ SELECT * FROM animals WHERE name != 'Gabumon';
...
+SELECT AVG(weight_kg) FROM animals;
+
+/* Who escapes the most, neutered or not neutered animals?*/
+SELECT neutered, MAX(escape_attempts) FROM animals GROUP BY (neutered);
+
+/* What is the minimum and maximum weight of each type of animal?*/
+SELECT species, MIN(weight_kg), MAX(weight_kg)FROM animals GROUP BY(species);
+
+/* What is the average number of escape attempts per animal type of those born between 1990 and 2000?*/
+SELECT species, AVG(escape_attempts) FROM animals WHERE date_of_birth BETWEEN '1990-01-01' AND '2000-12-31' GROUP BY(species);
```
Comment (mateo951) [inline queries.sql:87]: Transactions are on point 💣 Good Job!

**Review by mateo951 — APPROVED** [review]
### Hi @Jeanbulambo, 

Your project is complete!  There is nothing else to say other than... it's time to merge it 🐐 
### Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: data.sql (hunk 2 above)
Comment (Jeanbulambo) [reply data.sql:34]: thanks

**Pull request merged**